CHANGELOG.md

```diff
@@ -1,3 +1,211 @@
+# 5.0.0
+
+This version includes big changes around upgrading to selenium-webdriver 3.0.x.
+See the [selenium-webdriver changelog](https://github.com/SeleniumHQ/selenium/blob/master/javascript/node/selenium-webdriver/CHANGES.md). Some of the breaking change highlights are summarized below:
+
+- Minimum node version 6.9.x for selenium-webdriver dependency.
+- Jasmine typescript users will need to include the `@types/jasmine`. This
+  is to not cause conflicts with for mocha typings.
+- Removed `protractor.wrapDriver()`.
+- You can no longer use `repl` command from within `browser.pause()`. Instead,
+  use `browser.explore()` to directly enter the `repl`.
+- Methods removed: `WebDriver.prototype.isElementPresent`,
+  `WebElement.prototype.getRawId`, `WebElement.prototype.getInnerHtml`, and
+  `WebElement.prototype.getOuterHtml`.
+- Selenium-webdriver `ErrorCodes` have been deprecated.
+- Adding cookies have been changed from
+  `browser.manage().addCookie('testcookie', 'Jane-1234');` to
+  `browser.manage().addCookie({name:'testcookie', value: 'Jane-1234'});`
+- When testing with Firefox 47, use `marionette: false` to use legacy Firefox
+  driver.
+
+For the 5.0.0 release, we are still using the selenium standalone server 2.53.1
+and recommend using Firefox 47.
+
+## Features
```

Comment:
Some of the stuff from the quick list above should probably be formalized and moved to the breaking changes section.